tests/test_assets.py

```diff
@@ -843,21 +849,18 @@ def tearDownClass(cls):
     def test_len(self):
         """ Test the __len__ method of FutureChain.
         """
-        # None of the contracts have started yet.
-        cl = FutureChain(self.asset_finder, lambda: '2005-11-30', 'CL')
-        self.assertEqual(len(cl), 0)
-
-        # Sids 0, 1, & 2 have started, 3 has not yet started.
+        # Sids 0, 1, & 2 have started, 3 has not yet started, but all are in
+        # the chain
         cl = FutureChain(self.asset_finder, lambda: '2005-12-01', 'CL')
-        self.assertEqual(len(cl), 3)
+        self.assertEqual(len(cl), 4)
 
-        # Sid 0 is still valid its notice date.
+        # Sid 0 is still valid on its notice date.
         cl = FutureChain(self.asset_finder, lambda: '2005-12-20', 'CL')
-        self.assertEqual(len(cl), 3)
+        self.assertEqual(len(cl), 4)
 
         # Sid 0 is now invalid, leaving only Sids 1 & 2 valid.
         cl = FutureChain(self.asset_finder, lambda: '2005-12-21', 'CL')
-        self.assertEqual(len(cl), 2)
+        self.assertEqual(len(cl), 3)
```

Comment:
This comment makes it sound like there should only be 2 sids in the chain, should probably mention that 3 has not yet started.